src/LendingPool.sol

```diff
@@ -277,12 +268,12 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken {
         require(IVault(vault).decreaseMarginPosition(address(asset), transferAmount), "LP_R: Reverted");
```

Comment:
line 265: `_withdraw(transferAmount, address(0), vault);`

Reply:
Moved to separate issue